### ISSUE TITLE
`Unhandled exception in [StandaloneCoroutine{Cancelling}@5ef0e343, Dispatchers.IO]`

### ISSUE THREAD

**Issue by rakleed** [re-ovo/discord-ij#8]
One time WebStorm wouldn't start at all until I turned off Discord. Now I started with Discord enabled, but with the following error. JetBrains support replied that the problem is due to your plugin.

```log
  Unhandled exception in [StandaloneCoroutine{Cancelling}@5ef0e343, Dispatchers.IO]
  
  java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because the return value of "de.jcm.discordgamesdk.impl.commands.GetRelationships$Response.getRelationships()" is null
at de.jcm.discordgamesdk.RelationshipManager.lambda$new$5(RelationshipManager.java:53)
at de.jcm.discordgamesdk.Core.handleCommand(Core.java:329)
at de.jcm.discordgamesdk.Core.runCallbacks(Core.java:433)
at me.rerere.discordij.render.DiscordRPRender$init$1$1.invokeSuspend(DiscordRPRender.kt:31)
at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:100)
at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelled}@5ef0e343, Dispatchers.IO]
```

Environment:
```
WebStorm 2023.2.2
Build #WS-232.9921.42, built on September 12, 2023
Runtime version: 17.0.8+7-b1000.22 aarch64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 14.0
```

**Comment by re-ovo** [comment]
[discord-ij-1.0.9.zip](https://github.com/re-ovo/discord-ij/files/12883359/discord-ij-1.0.9.zip)
try this @rakleed

**Comment by re-ovo** [comment]
@rakleed hello?

**Comment by rakleed** [comment]
This problem does not occur to me every time I start WebStorm, but once every few days. It's okay for now.

**Comment by rakleed** [comment]
After installing your version and 1.0.10, the problem was not reproduced for me. Did you fix it in 1.0.10?

**Comment by rakleed** [comment]
@re-ovo after installing your version and 1.0.10, the problem was not reproduced for me. Did you fix it in 1.0.10?

**Comment by re-ovo** [comment]
> @re-ovo after installing your version and 1.0.10, the problem was not reproduced for me. Did you fix it in 1.0.10?

Maybe, Does the function work properly?

**Comment by rakleed** [comment]
After installing 1.0.10, the IDE did not crash with an error on startup.
